nuxeo-customer-project-sample-package/src/main/assemble/assembly.xml

```diff
@@ -63,86 +64,85 @@
           <artifact type="pom" />
           <artifact groupId="nuxeo-studio" />
           <!-- Filter out your artifacts using their groupId for instance -->
-          <artifact groupId="com.customer.sample*"/>
+          <artifact groupId="com.customer.sample*" />
         </excludes>
       </artifact:set>
     </copy>
-    <nx:rmdups dir="${outdir}/nxr/lib"/>
+    <nx:rmdups dir="${outdir}/nxr/lib" />
     <!-- Temporary ZIP Nuxeo NXR to compare with base distribution NXR -->
-    <zip destfile="${outdir}/nxr-${maven.project.version}.zip" basedir="${outdir}/nxr"/>
-    <!-- Optionally attach the NXR for later use by other modules -->
-    <artifact:attach file="${outdir}/nxr-${maven.project.version}.zip" classifier="nxr" type="zip"/>
+    <zip destfile="${outdir}/nxr-${maven.project.version}.zip" basedir="${outdir}/nxr" />
 
     <!-- Your MP only needs to include files not already provided by the base distribution -->
     <nx:zipdiff
-      file1="${outdir}/nxr-${maven.project.version}.zip"
-      file2="${outdir}/nuxeo-nxr-server-${nuxeo.distribution.version}.zip"
-      includesfile="${outdir}/includes"
-      excludesfile="${outdir}/excludesfile"
-      patternsetid="nxr.versus.distribution"
-      ignoreContent="false"
-      ignoreContentPattern=".*\.jar$"></nx:zipdiff>
+            file1="${outdir}/nxr-${maven.project.version}.zip"
+            file2="${outdir}/nuxeo-nxr-server-${nuxeo.platform.version}.zip"
+            includesfile="${outdir}/includes"
+            excludesfile="${outdir}/excludesfile"
+            patternsetid="nxr.versus.distribution"
+            ignoreContent="false"
+            ignoreContentPattern=".*\.jar$"></nx:zipdiff>
     <unzip src="${outdir}/nxr-${maven.project.version}.zip" dest="${outdir}/marketplace/install">
-      <patternset refid="nxr.versus.distribution"/>
+      <patternset refid="nxr.versus.distribution" />
```

Comment:
slight diff with https://github.com/nuxeo/nuxeo-explorer/blob/master/packages/src/main/assemble/assembly.xml#L62-L71, not sure what's the right thing to do